### ISSUE TITLE
User report: 1050 Ti

### ISSUE THREAD

**Issue by 949f45ac** [tiagoshibata/meth-pills#4]
Hey there,

first off, thanks a lot for creating an open source version of this!

Running it for my ol’ 1050 Ti.

Stock:
- kawpow: 6 MH/s
- autolykos2: 27 MH/s

Safe timings:
- kawpow: basically unchanged
- autolykos2: 35 MH/s

Normal timings (no `-s`):
- autolykos2: 36 MH/s

The safe timings definitely have a good impact on this card already.
Going beyond safe timings does not give much improvement.

Cheers!

**Comment by 949f45ac** [comment]
Ah, I did not measure power usage, NVML does not report power for this card.
But from looking at temperature/fan, I would assume the timings do not lead to significantly higher power consumption.
(The card’s power cap is 75W, it does not even have a 6-Pin connector, it gets all power through the PCIe slot. I have set the power cap to 52.5W via nvidia-smi (no clue if this gets respected by the card) without an impact on hashrate.)
